SU-HSDO/suhumsci
# Fix js error in select-list library

# READY FOR REVIEW

## Summary
- Fix js error that was breaking the select list widget

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Check the following pages. For all of them, confirm that the select (both single and multiple) work as expected and that there are no JS errors in the browser console:
    - https://politicalscience.suhumsci.loc/undergraduate-program/current-courses
    - https://economics.suhumsci.loc/courses
    - https://art.suhumsci.loc/academics/courses
    - https://english.suhumsci.loc/courses
    - https://linguistics.suhumsci.loc/courses
    - https://aaas.suhumsci.loc/academics/courses

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
